modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -116,13 +116,10 @@ func (k Keeper) GetActiveChannelID(ctx sdk.Context, portID string) (string, bool
 
 // HasActiveChannel retrieves the active channelID from the store key by the provided portID & checks if the channel in question is in state OPEN
 func (k Keeper) HasActiveChannel(ctx sdk.Context, portID string) (string, bool) {
-	store := ctx.KVStore(k.storeKey)
-	key := icatypes.KeyActiveChannel(portID)
-
-	channelID := string(store.Get(key))
+	channelID, _ := k.GetActiveChannelID(ctx, portID)
```

Comment:
I think you should still check if the active channel mapping exists instead of querying with a blank channel id

Reply:
I think its probably considered best practice to check the bool anyway and not discard return values with `_`

Reply:
Updated